_rules/inline-link-in-paragraph-has-different-background-color-and-distinguishable-style-on-hover-and-focus-2803b8.md

```diff
@@ -0,0 +1,143 @@
+---
+id: 2803b8
+name: Inline link has different background color and distinguishable style on hover and focus
+rule_type: atomic
+description: |
+  This rule checks that inline links with different background color have a style on hover and focus that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: The element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: The element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][].
+
```

Comment:
This applicability does not match the description. Surely the link text should also have a background colour that is in someway different to the background colour of the non-link line text.

Reply:
The issue here is that this is an atomic rule of a composite. Per the ACT-Rules definition, the applicability of the composite is the union of the applicability of the atomic rules. Therefore, if we changed the applicability of this one (I agree that to match the description that should be the case) then we would have to change the applicability of the composite, which we want to apply to inline links in general.
So, once again, we end up with something in the expectation that could have been in the applicability.

Reply:
Then perhaps the description needs to change.

Reply:
Can you check the one I just updated?